docs/PC-8800.md

```diff
@@ -44,40 +25,171 @@ At this link you can find information on the BIOS files of the emulator:
 | N88EXT2.ROM | B7C8BCEA219B77D9CC3EE0EFAFE343CC307425D1 | AF2B6EFA | 8 KB |
 | N88EXT3.ROM | EFCE0B51CAB9F0DA6CF68507757F1245A2867A72 | 7713C519 | 8 KB |
 
-If you add these 2 files too, you will have the translation of the menus in kanji characters:
+Adding the next 2 files :
+
+Menus (Quasi88 only) and Japanese games will display the Japanese kanji characters. [Quasi88, lr-quasi88]
+
+Not adding the next 2 files :
+
+Menus will be in English (Quasi88 only) and Japanese games will **not** display the Japanese kanji characters. [Quasi88, lr-quasi88]
 
 | File | SHA1 | CRC32 | Size |
 | :--: | :--: | :--: | :--: |
 | N88KNJ1.ROM | 82E11A177AF6A5091DD67F50A2F4BAFDA84D6556 | 6178BD43 | 128 KB |
 | N88KNJ2.ROM | 7E6591CD465CBB35D6D3446C5A83B46D30FAFE95 | 154803CC | 128 KB |
 
-Put the bios files in lowercase otherwise the emulator won't find them, and result in black screen.
 
-## Controls
+## Emulator: [Quasi88](https://www.eonet.ne.jp/~showtime/quasi88/)
+
+Note that the emulator (and most documentation) is written in Japanese.
+
+This emulator (if available) can be installed from the experimental menu of the [RetroPie-Setup script.](Updating-RetroPie#using-the-retropie-setup-script)
+
+This emulator has better keyboard support. 
+
+Recommended for :
+
+- Working in "BASIC", "DOS" or "CP-M"
+- Editing disks or files
+
+## BIOS [Quasi88]
+
+Place your BIOS files in:
+```
+/home/pi/RetroPie/BIOS/pc88
+```
+
+## ROMS [Quasi88]
+
+Accepted File Extensions: **.d88 (disk) .88d (disk) .cmt (tape) .t88 (tape)** 
+
+Place your PC-88 ROMs in:
+```
+/home/pi/RetroPie/roms/pc88
+```
+
+## Controls [Quasi88]
+
 By default, Quasi88 uses the keyboard for controls.  To enable a controller:
 
 ```
-Press F12.  This will bring up a menu with several tabs at the top.
+Press "F12".  This will bring up a menu with several tabs at the top.
 Click the "mouse" tab.
 Click the dropdown that's labelled "nomouse". Change it to "joystick".
 ```
 
 Once this is done, the controls will be automatically mapped as such:
 ```
 Left Joystick - Number Pad
-A Button - Z key
-B button - X key
+A Button - "Z key"
+B button - "X key"
+```
```

Comment:
Please use a simple list to show the controls:
* *A* button - Z key
...

It's also not clear what side is the RetroPad and what is the machine input - does the A button on the gamepad simulates the Z key in the emulated machine or vice-verse ?